SpatialGDK/Documentation/content/get-started/gdk-and-starter-project.md

```diff
@@ -0,0 +1,70 @@
+# Get started: 3 - Set up the Unreal GDK module and Starter Project
+
+Follow the steps below to:
+
+* Clone the Starter Project repository.
+* Clone the Unreal GDK into the `Plugins` folder of your game.
+* Build the Unreal GDK module dependencies which the Starter Project needs so it can work with the GDK.
```

Comment:
Keep the step snappy so it works as a title below...(maybe use a line break and brackets if you want to keep it in)

* Build the Unreal GDK module dependencies
 <br/>(the Starter Project needs these so it can work with the GDK)